daemon/config/README.md

```diff
@@ -141,8 +141,76 @@ External plugins will have only 2 options at `netdata.conf`:
 
 External plugins that need additional configuration may support a dedicated file in `/etc/netdata`. Check their documentation.
 
-### Per chart configuration
+## Per-chart configuration
+
+In this area of `netdata.conf` you can find configuration options for individual charts. They appear in sections
+following the pattern `[NAME]`.
+
+Using the settings and values under these sections, you can control all aspects of a specific chart. You can change its
+title, make it appear higher in Netdata's [menu](../../web/gui/README.md#menus), tweak its dimensions, and much more.
+
+To find the name of a given chart, and thus the name of its section in `netdata.conf`, look at the top-left corner of a
+chart:
+
+![Finding the unique ID of a
+chart](https://user-images.githubusercontent.com/1153921/67443082-43b16e80-f5b8-11e9-8d33-d6ee052c6678.png)
+
+Every per-chart configuration section has several common settings, which are listed in the table just below. Beneath
+that is information about lines that begin with `dim`, which affect a chart's dimensions.
+
+| Setting           | Function                                                                                                                                                                                                                                          |
+| :---------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `history`         | Change the number of metrics stored for this chart when using the `save` memory mode. Should be less than or equal to the global `history` setting.                                                                                               |
```

Comment:
This is valid for memory mode other than `dbengine`

Reply:
Got it. Fixed this!